pkgs/development/python-modules/datadiff/default.nix

```diff
@@ -2,24 +2,35 @@
 , buildPythonPackage
 , fetchPypi
 , nose
+, pythonOlder
+, six
 }:
 
 buildPythonPackage rec {
   pname = "datadiff";
-  version = "1.1.6";
+  version = "2.0.0";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
 
   src = fetchPypi {
     inherit pname version;
-    extension = "zip";
-    sha256 = "f1402701063998f6a70609789aae8dc05703f3ad0a34882f6199653654c55543";
+    hash = "sha256-VASUQKiW3lJH7i57eqVCX5OayDLDftshAfjTA26Jyqg=";
   };
 
-  buildInputs = [ nose ];
+  checkInputs = [
+    nose
+    six
+  ];
```

Comment:
`checkPhase` should be specified unless `pytestCheckHook` is in `checkInputs`

Reply:
Good to know. Thanks for the hint.

Actually the tests are no part of the releases available on PyPI and upstream pretty much abandoned it.